Casks/graalvm-ce-java11.rb

```diff
@@ -32,9 +32,9 @@
     Installing GraalVM CE (Java 11) in #{jvms_dir} requires root permissions.
     You may be asked to enter your password to proceed.
 
-    On macOS Catalina, you may get a warning that "the developer cannot be
-    verified". This check can be disabled in the "Security & Privacy"
-    preferences pane or by running the following command:
+    On macOS Catalina or later, you may get a warning when you open the GraalVM
```

Comment:
Not sure about "open". Also, is it a warning? A warning generally means that something you are trying to do might have adverse consequences. I thought macOS just refused point blank to run GraalVM unless you remove the quarantine attribute.

Reply:
Right, I've decided to use the word "use" instead of "open". I think you still get a warning, on the latest macOS it just sounds very dangerous ("could damage your system" or something like that).